binary/expression.proto

```diff
@@ -68,6 +77,18 @@ message Expression {
             repeated KeyValue key_values = 1;
         }
 
+        message Decimal {
+            // little-endian twos-complement integer representation of complete value (ignoring precision)
+            bytes value = 1;
+            
+            // declared scale of decimal literal
+            int32 scale = 2;
+            
+            // place to put decimal within twos-complement value.
+            int32 precision = 3;
```

Comment:
:sweat_smile: I hit this (missing field) recently, but forgot to open a PR to fix it.

Reply:
Yeah, someone else mentioned it. I've taken all literal changes out of this pr. We can capture separately.